### PR TITLE
chore: [#186393974] correct padding on profile side nav header

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

https://www.pivotaltracker.com/story/show/186393974

### Approach
I had to remove some padding from the header, additionally I ended up doing some re-factoring to make the code a lot of bit easy to read, and a little bit more efficient for when we build the storybook component

## Code author checklist

- [ ] I have performed a self-review of my code
- [ ] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [ ] I have not used any relative imports
- [ ] I have checked for and removed instances of unused config from CMS
- [ ] I have pruned any instances of unused code
- [ ] I have not added any markdown to labels, titles and button text in config
- [ ] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [ ] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

**Review by AnJuHyppolite — COMMENTED** [review]
Save for my two comments, it looks good. Nice, Faraz!

File: web/src/components/profile/ProfileTabNav.tsx [hunk 1]
```diff
@@ -7,6 +7,11 @@ import { BusinessPersona } from "@businessnjgovnavigator/shared/profileData";
 import { Business } from "@businessnjgovnavigator/shared/userData";
 import { ReactElement } from "react";
 
+const INFO_TAB = "info";
+const NUMBERS_TAB = "numbers";
+const DOCUMENTS_TAB = "documents";
+const NOTES_TAB = "notes";
```
[the remark this answers is not in the document]
Comment (FarazA22) [reply web/src/components/profile/ProfileTabNav.tsx:13]: It's common to use variables in all caps, often with underscores to indicate the value is hardcoded and will not change.

Looking into this a bit further, we don't use this convention as frequently as I thought. I'll change it.

File: web/src/components/profile/ProfileTab.tsx [hunk 2]
```diff
@@ -1,38 +1,23 @@
 import { Icon } from "@/components/njwds/Icon";
-import { useConfig } from "@/lib/data-hooks/useConfig";
 import { ProfileTabs } from "@/lib/types/types";
 import { ReactElement } from "react";
 
 interface Props {
   tab: ProfileTabs;
   activeTab: ProfileTabs;
   setProfileTab: (profileTab: ProfileTabs) => void;
+  children: string;
```
Comment (jphechter) [inline web/src/components/profile/ProfileTab.tsx:9]: [pebble] I don't like calling a single string `children`. This seems like an inconsistent use of a prop that is used commonly in React. Additionally, this is a plural name that is representing a single value.

File: content/src/fieldConfig/profile.json [hunk 3]
```diff
@@ -370,13 +370,13 @@
...
       "backButtonText": "Cancel",
-      "pageTitle": "### Business Profile",
+      "pageTitle": "Business Profile",
       "escapeModalEscape": "Go back",
       "escapeModalBody": "You have unsaved changes that will be lost.",
       "standingDocFileTitle": "Certificate of Good Standing",
       "profileErrorAlertOneField": "the field",
       "profileTabDocsTitle": "Reference Documents",
-      "profileTabRefTitle": "Reference Numbers",
+      "profileTabNumbersTitle": "Reference Numbers",
```
[the remark this answers is not in the document]
Comment (FarazA22) [reply content/src/fieldConfig/profile.json:379]: The other variable names, like `profileTabDocsTitle` don't include the word reference. Do you think all the variable names should be updated?

File: web/src/components/profile/ProfileTab.tsx (hunk 2 above)
Comment (FarazA22) [reply web/src/components/profile/ProfileTab.tsx:9]: updated to tabText

File: content/src/fieldConfig/profile.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnJuHyppolite) [reply content/src/fieldConfig/profile.json:379]: That's one reason why I suggested `profileTabNumsTitle` vs spelling Numbers out, but if we're going to be explicit and write out Numbers, we should be explicit with the others.

**Pull request merged**